native/core/Cargo.toml

```diff
@@ -77,6 +77,7 @@ datafusion-comet-proto = { workspace = true }
 object_store = { workspace = true }
 url = { workspace = true }
 chrono = { workspace = true }
+datafusion-objectstore-hdfs = { git = "https://github.com/comphead/datafusion-objectstore-hdfs", branch = "master", optional = true }
```

Comment:
@andygrove I'm keeping the updated HDFS object storage in personal repo for now, let me know if there any concerns

Reply:
Is there an expected timeline for when we can move to an official release? Meantime, since we have pointed to a personal repo in the past, it is reasonable to do so for this as well (especially since this is already behind some configuration flags).

Reply:
Will be addressed in #1368

Reply:
Done